docs/src/tool/index.md

```diff
@@ -27,44 +27,77 @@ You can copy this file to other repositories or move it to a directory in your `
 If you're using Windows, consult [the guide to using Reach on Windows](##guide-windows).
 :::
 
-# {#ref-usage} Select a version
+# {#ref-usage} Usage
```

Comment:
I don't like the three levels of headings. I think it looks better on the page to just have one heading level the entire time, even though it is less ontologically pure